src/compiler/variables.jl

```diff
@@ -71,12 +71,12 @@ function toqubo_variables!(model::VirtualQUBOModel{T}, ::AbstractArchitecture) w
         end
     end
 
-    # -*- Discretize Real Ones 🤔 -*-
+    # Discretize Real Ones 🤔 
     for (x, (a, b)) in ℝ
         if isnothing(a) || isnothing(b)
             error("Unbounded variable $(x) ∈ ℝ")
         else
-            # TODO: Solve this bit-guessing magic???
+            # TODO: Solve this bit-guessng magic???
```

Comment:
You added a typo here and more importantly, move this to an issue instead of it being buried in the code

Reply:
This was actually solved for good. The idea below this comment was implemented and seems to work well.
This is about to be moved to the docs.

Reply:
Maybe yes

Reply:
Added to #63